app/src/main/res/layout/main_fragment.xml

```diff
@@ -326,7 +326,7 @@
             android:text="@string/navbar_create_puzzle_button"
             android:textAppearance="@style/TextAppearance.AppCompat.Large"
             android:textColor="@color/customBlack"
-            app:backgroundTint="@color/creamBackground" />
+            app:backgroundTint="#D8CDAC86" />
```

Comment:
Keep this as it were.  Better to use colors.xml than to specify a color in the layout XML.